src/main/resources/com/google/api/codegen/java/sample.snip

```diff
@@ -64,6 +62,9 @@
       @end
 
       {@class.requestBodyTypeName} {@class.requestBodyVarName} = new {@class.requestBodyTypeName}();
+      @join field : class.requestBodyFields if class.requestBodyFields
```

Comment:
Okay, thanks for clarifying. I had thought it looked like a comprehension syntax.